_posts/2020-09-15-solidity-underhanded-contest.md

```diff
@@ -60,7 +60,7 @@ Furthermore, the winners and all qualified submissions will receive a custom NFT
 
 Consider the following aspects and properties for your submissions:
 
-+ **Simplicity & Length**: Since it’s much easier to hide a vulnerability in complex and poorly written code, short and clean submissions will be scored higher than those that are lengthy and complicated.
++ **Simplicity & Briefness**: Since it’s much easier to hide a vulnerability in complex and poorly written code, short and clean submissions will be scored higher than those that are lengthy and complicated.
```

Comment:
```suggestion
+ **Simplicity & Briefness**: Since it's much easier to hide a vulnerability in complex and poorly written code, short and clean submissions will be scored higher than those that are lengthy and complicated.
```